styles.css

```
@@ -0,0 +1,77 @@
+body {
+    background: radial-gradient(rgb(100, 236, 129), rgba(250 254 253));
+  }
+  
+  main {
+    background: radial-gradient(
+      rgba(52, 130, 231, 0.6),
+      rgba(197, 197, 209, 0.6)
+    );
+    width: 800px;
+    height: 650px;
+    margin: 40px auto auto auto;
+    box-shadow: 10px 10px 5px #d7e0df;
+    display: block;
+    border-radius: 10px;
+  }
+  
+  h1 {
+    padding: 20px 20px 0px 20px;
```

Având în vedere că în stânga și în dreapta e aceeași distanță, se poate lăsa varianta cu 3 valori:

```suggestion
    padding: 20px 20px 0;
```

Am scos și unitatea de măsură atunci când avem 0